src/app/utils/number-utils.ts

```diff
@@ -0,0 +1,19 @@
+import BigNumber from 'bignumber.js'
+
+export function fromBaseUnits(valueInBaseUnits: string, decimals: number): string {
+  const value = new BigNumber(valueInBaseUnits).shiftedBy(-decimals) // / 10 ** decimals
+  if (value.isNaN()) {
+    throw new Error(`Not a number in fromBaseUnits(${valueInBaseUnits})`)
+  }
+  return value.toFixed()
+}
+
+export const getGasPrice = ({ fee = '', gasUsed = '' }): string | null => {
+  if (!fee || !gasUsed || gasUsed === '0') {
+    return null
+  }
+
+  return BigNumber(fee).div(BigNumber(gasUsed)).toFixed()
+}
+
+export const fromBaseUnitsToNano = (wei: string): string => fromBaseUnits(wei, -9)
```

Comment:
This is not taking in base units

Reply:
True. But what would wei unit be called then?

Reply:
Thinking about it now, you probably meant the function parameter, not the naming. Nevermind.

Reply:
> `export const fromBaseUnitsToNano = (valueInBaseUnits: string): string => fromBaseUnits(valueInBaseUnits, -9)`

Both. This takes in values like "0.0000001", clearly not in base units

Reply:
Ah right, for some reason I though on Sapphire 1 ROSE = baseUnit 🤦 

Well should I just name it `unit` then? Or maybe `exaBaseUnit` - but that one is also not correct, as it depends on the ParaTime?

Reply:
Did generalize, I hope it's ok now.

Reply:
:D 
(maybe toBaseUnits)

Reply:
`toBaseUnit` would imply that "conversion" is done by 10^18, not by 10^9. Either way, I am keeping it as is.